docs/_themes/clusterhq/layout_docs.html

```diff
@@ -2,6 +2,10 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    {% if pagename == 'error_pages/404' %}
+    <!-- The 404 error page may be served at any depth in the URL hierarchy. -->
+    <base href="/en/latest/error_pages/404.html">
```

Comment:
I think it doesn't matter, but it may be worth a comment: When publishing a dev release to the staging bucket, this base-ref will still refer to the latest marketing release.